it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureControllerISpec.scala

```diff
@@ -23,13 +23,66 @@ class CaptureBox5FigureControllerISpec extends ComponentSpecHelper with CaptureB
 
   "POST /box-5-figure" should {
     "redirect to CaptureLastMonthSubmitted" in {
-      lazy val result = post("/box-5-figure")()
+      lazy val result = post("/box-5-figure")(
+        "box5_figure" -> "1234.56"
+      )
 
       result must have(
         httpStatus(SEE_OTHER),
         redirectUri(routes.CaptureLastMonthSubmittedController.show().url)
       )
     }
+
+    "return a BAD_REQUEST if the box 5 figure is missing" in {
+      lazy val result = post("/box-5-figure")(
+        "box5_figure" -> ""
+      )
+
+      result.status mustBe BAD_REQUEST
+    }
+
+    "return a BAD_REQUEST if the box 5 figure has invalid format" in {
+      lazy val result = post("/box-5-figure")(
+        "box5_figure" -> "1234.5"
+      )
+
+      result.status mustBe BAD_REQUEST
+    }
+
+    "return a BAD_REQUEST if the box 5 figure has invalid characters" in {
+      lazy val result = post("/box-5-figure")(
+        "box5_figure" -> "1234.oo"
+      )
+
+      result.status mustBe BAD_REQUEST
+    }
+
+    "redirect to CaptureLastMonthSubmitted if the box 5 figure has a correct value " in {
```

Comment:
Sorry, could you make this a negative value to prove that it submits correctly please?